share/man/man1/brew.1

```diff
@@ -554,10 +554,6 @@ HOMEBREW_BROWSER
 If set, uses this setting as the browser when opening project homepages, instead of the OS default browser\.
 .
 .TP
-HOMEBREW_BUILD_FROM_SOURCE
```

Comment:
It's impossible to stop people filing issues. But we can establish some regulation that when a unsupported feature relate issue is filed, we will:
* if user has PR to fix it, we will review it. 
* if it's extrem easy to fix for us, we will fix it. 
* if it's not easy to fix but be related to more substantial supported feature, it will be treated as normal issue. 
* if non of above applied, we will tag wontfix, close it and encourage user to fix on their own if they really need it.